tests/tests.py

```diff
@@ -1,48 +1,61 @@
-from django.urls import reverse
+from django.apps import apps
+from django.db.models import Case, Sum, When, F, FloatField, ExpressionWrapper
+from django.db.models import Count
 from django.test import TestCase
-from django.test.utils import setup_test_environment
 from django.utils.encoding import smart_text
 
-
-from django.db.models import F, Count, Min, Max, Sum, Value, Avg
-from data_interrogator.interrogators import Interrogator, Allowable
-from django.apps import apps
-from django.db.models import Case, Lookup, Sum, Transform, Q, When, F, FloatField, ExpressionWrapper
 from data_interrogator import exceptions
+from data_interrogator.interrogators import Interrogator, Allowable
 
 
 class TestInterrogatorPages(TestCase):
-    fixtures = ['data.json',]
-    
+    """NB: these tests use the url configuration specified by the shop display app"""
+    fixtures = ['data.json']
+
     def test_page_room(self):
+        """Test that the interrogator page for sales people returns the correct results"""
+
         response = self.client.get(
-            "/data/?lead_base_model=shop%3Asalesperson&"
-            "filter_by=&filter_by=&"
-            "columns="
+            '/full_report/?lead_base_model=shop%3Asalesperson'
+            '&filter_by='
+            '&columns='
             "name||"
             "sum%28sale.sale_price+-+sale.product.cost_price%29||"
-            "count%28sale%29&sort_by="
+            '&sort_by=&action='
```

Comment:
Would be so much cleaner to use `urlencode` with a dict here, wouldn't need all the % stuff

Reply:
Yeah, agreed, will fix it

Reply:
Fixed